CHANGELOG.md

```diff
@@ -4,6 +4,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Added
+
+- Support all platforms lacking 64 bit atomics.
```

Comment:
```suggestion
- Support all platforms with 32 bit atomics lacking 64 bit atomics.
```

Assuming there are non-u32 atomics platforms as well.

Reply:
Yeaaaah, I kinda implicitly left those out of the whole equation because you need _something_, but I can clarify the changelog !